src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/mapper/OffenderToAssessmentSummaryMapper.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.mapper
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity.Offender
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.exception.PrisonerNotFoundException
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.AssessmentSummary
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.TaskProgress
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.toSummary
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.prison.PrisonService
+
+@Component
+class OffenderToAssessmentSummaryMapper(
+  private val prisonService: PrisonService,
+) {
+
+  fun map(offender: Offender): AssessmentSummary {
+    val currentAssessment = offender.currentAssessment()
+    val prisonerSearchResults = prisonService.searchPrisonersByNomisIds(listOf(offender.prisonNumber))
```

Comment:
It'd be good to bulk call this rather than one per prisoner

Reply:
or is this currently _only_ used for getting a single prisoners details?

Reply:
I think it's only for one!?